indy_common/authorize/auth_request_validator.py

```diff
@@ -21,11 +22,13 @@ def validate(self, request, action_list: [AbstractAuthAction]):
 
 
 class WriteRequestValidator(AbstractRequestValidator, CompositeAuthorizer):
-    def __init__(self, config, auth_map, cache):
+    def __init__(self, config, auth_map, cache, anyone_can_write_map=None):
```

Comment:
I'm not sure that we need `anyone_can_write_map` anymore with the new approach.
Our new auth constraints assume to have the following format:
`(TRUST_ANCHOR, fees=0 ) OR (ANYONE, fees=10).`
So, by default a txn will succeed only when the first constraint is True.
Plugins can register a validator for fees, so that the can validate that the second condition is true.